### DISCUSSION TITLE
How to stop ghelper opening

### DISCUSSION THREAD

**Discussion by Randomlygeneratedusername69** [seerge/g-helper#1560] [Q&A]
When I press the ROG key G-Helper opens. id prefer it if this didn't happen.

Under key binds only f2,f3,f4,fn+c appear on the list. ROG key bind isn't there for me like it is in the screenshots.

I'm on the Zephyrus duo 16 GX650RW. anyone have any ideas on how to fix this? I've tried reinstalling, installing previous releases, a full pc reset.

**Comment by seerge** [comment]
@Randomlygeneratedusername69 this build should show rog key in bindings for Duo

[GHelper.zip](https://github.com/seerge/g-helper/files/13254540/GHelper.zip)

**Reply by fscottcopeland** [reply]
Any chance this can be an option? I found it quite useful to use the ROG key to open GHelper!

**Reply by seerge** [reply]
Option where ? It's already added in a built linked here.

**Comment by Randomlygeneratedusername69** [comment]
Thankyou, i really appreciate this
